### ISSUE TITLE
how to increase FPS of  custom model which i was trained on yolov4mish/csp ?

### ISSUE THREAD

**Issue by akashAD98** [AlexeyAB/darknet#7690]
i have a weight file that has a size of 379MB. I want to increase the FPS or inference of my model. apart from using GPU & half_preseion what are different ways in which I can increase model FPS without affecting its accuracy.

I thought to convert my model weight into TensorFlow/TF.lite /tensorRT /pytorch .pth model .& let's compare the model performance. Even there is no any source code for converting mish/CSP model into .pb & .pth format. I m really grateful if anyone can help me with this issue.

**Comment by poornimajd** [comment]
May be you can try tkdnn. Also try to decrease width and height in cfg file,it will affect the accuracy but increase in fps will not really be significant without much decrease in accuracy.
I think model pruning may also be one way to explore.

**Comment by poornimajd** [comment]
Yeah possibly.Did you try with tkdnn? Or may be if you could try some other model like yolov4 without mish/csp and if this works well ,then the problem is something else.

**Comment by bulatnv** [comment]
@akashAD98  
Convert model to TensorRT via Onnx. This definitely works.

**Comment by akashAD98** [comment]
@bulatnv @poornimajd thank you. i will try TensorRT

**Comment by matt-sharp** [comment]
Or you can try using OpenCV DNN.
